src/worker-helpers.js

```diff
@@ -13,6 +13,12 @@ const Cache = {
   LAST_MODULES_PATH: null
 }
 
+const cleanPath = (path) => {
+  const unTildeify = fs.normalize(path)
+  const unEnvVarify = resolveEnv(unTildeify)
```

Comment:
__*Preference*__

Having an ending of `ify` makes me think that these variables are functions, but it seems that they are actually just strings.  Honestly, I think you could just combine these three lines into:

```js
return resolveEnv(fs.normalize(path));
```

Reply:
I think I was thinking of `ified`.